include/nuttx/fs/ioctl.h

```diff
@@ -235,6 +235,16 @@
                                            * OUT: Current file xip base address
                                            */
 
+#define FIOC_GETFLAGS       _FIOC(0x0016) /* IN:  None
+                                           * OUT: None
+                                           */
+#define FIOC_SETFLAGS       _FIOC(0x0017) /* IN:  The flags that need to set to file
+                                           * OUT: None
+                                           */
+#define FIOGCLEX            _FIOC(0x0018) /* IN:  FAR int *
```

Comment:
FIOC_ prefix

Reply:
follow FIOCLEX,FIONCLEX, FIOGCLEX is used to get flags.